### ISSUE TITLE
Sign up timeout error (500)

### ISSUE THREAD

**Issue by StijnJvT** [Codewars/codewars.com#1631]
I complete the initial kata (python), then when prompted to sign up I put in email, username and password, and when submitting, after a while of no response I get the timeout error 500, server taking more than 45 seconds to respond.

I tried this on two separate systems, at various times during the past week, using both firefox and chrome.

Probably relevant extra info: I just managed to sign up using a different user name, without problems, right after not succeeding with my original choice. I suspect this might something to do with signing up with an already taken username?

**Comment by kazk** [comment]
Sorry for the inconvenience.
This is a known issue that happens very _inconsistently_. Some days we have more than 1000 users register without a problem.
I'm going to close this as duplicate of many others.

---

> I just managed to sign up using a different user name, without problems, right after not succeeding with my original choice. I suspect this might something to do with signing up with an already taken username?

What was your first choice? I'll check if it's taken.

**Comment by StijnJvT** [comment]
No worries, thanks for replying. My first choice of username was "Stijn". Just fyi, I tried this at different times, on different systems, on different browsers, at least 15 times, over the course of a week. The behavior seemed pretty consistent to me.

I am happy for you to close this as a duplicate -- I only saw a related thread of a few weeks old, so figured that had probably already been resolved..
